ignite/engine/__init__.py

```diff
@@ -1,4 +1,4 @@
-from ignite.engines.engine import Engine, Events, State
+from .engine import Engine, State, Events
```

Comment:
updated. Looks ugly as hell now 😂

Reply:
> from ignite.engine.engine import Engine
we should have called the package engine to complete the similarity :)

Otherwise, if you put `Engine`, `State`, `Event` into `ignite.__init__.py` it could be nicer, what do you think ?